spec/system/page_objects/pages/custom_sections_sidebar.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Pages
+    class CustomSectionsSidebar < PageObjects::Pages::Base
```

Comment:
I don't think this should be a page but should instead be a component.

Reply:
Perhaps the methods here should just be under `PageObjects::Components::Sidebar` instead? I don't really think we need another component here when interacting with the sidebar.